server/controllers/card.js

```diff
@@ -0,0 +1,77 @@
+const asyncHandler = require("express-async-handler");
+const Card = require("../models/Card");
+const { v4: uuidv4, v4 } = require("uuid");
+
+exports.getCards = asyncHandler(async (req, res) => {
+  const Cards = await Card.find();
+
+  return res.json(Cards);
+});
+
+exports.getCard = asyncHandler(async (req, res) => {
+  try {
+    const Card = await Card.findById(req.params.id);
+
+    if (!Card) res.status(404).json({ error: "Card not found" });
+
+    return res.status(200).json(Card);
+  } catch (error) {
+    return res.json(error);
+  }
+});
+
+exports.createCard = asyncHandler(async (req, res) => {
+  const { name, tag, columnId } = req.body;
+  const newCard = await Card.create({
+    name: name,
+    tag: tag,
+    columnId: columnId,
+    _id: v4(),
+    createdAt: Date.now(),
+  });
+
+  return res.status(200).json({ Card: newCard });
+});
+
+exports.updateCard = asyncHandler(async (req, res, next) => {
+  const {
+    _id,
+    name,
+    tag,
+    description,
+    deadline,
+    comment,
+    attachment,
+    checklist,
+    columnId,
+    index,
+    createdAt,
```

Comment:
Typically createdAt dates aren't changeable, however an `updatedAt` is generally preferred.

Reply:
Worth mentioning with timestamps we don't even need to worry about `updateAt` that will be handled automatically.

Reply:
So with timestamps enabled, do we not need either of these in the Model either? Or do you just mean we don't need to pass them to the update method in the controller?

Reply:
WIth timestamps enabled, we do not need to expect the user to update the `createdAt` field. So you can remove the createdAt from the models and where you are destructuring it from the incoming request body.

Reply:
Mongo will handle those fields and automatically inject them in our documents and when we update the document it will automatically set the updateAt field for us. Which is great, but yeah.